.github/workflows/cd.yml

```diff
@@ -1,6 +1,6 @@
-name: React Template CD
+name: ITUNES React CD
 on:
-  push:
+  pull_request:
```

Comment:
You dont need this

Reply:
yeah when I'm done with testing I will change it to push for the master